.github/workflows/ci_benchmark.yml

```
@@ -0,0 +1,150 @@
+### Inspired from https://github.com/scikit-image/scikit-image/blob/main/.github/workflows/benchmarks.yml
+
+name: Benchmark
+
+on:
+  pull_request:
+    types: [labeled, synchronize]
+
```

We probably want auto-cancel (in case someone is spam pushing to a PR) and restrict permission?
```suggestion

concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true

permissions:
  contents: read

```